webapp/loggedin/surveyViews/record/recordView.js

```diff
@@ -54,6 +56,10 @@ const RecordView = props => {
       alert(i18n.t('recordView.justDeleted'))
       recordDeleted(history)
     })
+    AppWebSocket.on(WebSocketEvents.recordEditTimeout, () => {
+      alert(i18n.t('recordView.editTimeout'))
```

Comment:
please use showAppNotification and not alert / same for recordDeleted

Reply:
Ok